# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/client/netmonitor/test/browser_net_copy_as_powershell.js
# Commit: 99e71b905661
# Full Hash: 99e71b9056618a081f8bc2fb7803ba208db3a5b0
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-10 09:45:41
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin,devtools-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 68f119112940 -r 99e71b905661 devtools/client/netmonitor/test/browser_net_copy_as_powershell.js
--- a/devtools/client/netmonitor/test/browser_net_copy_as_powershell.js	Tue Apr 09 15:26:45 2024 +0000
+++ b/devtools/client/netmonitor/test/browser_net_copy_as_powershell.js	Tue Apr 09 15:26:46 2024 +0000
@@ -19,7 +19,7 @@
 -Headers @{
 "Accept" = "*/*"
   "Accept-Language" = "en-US"
-  "Accept-Encoding" = "gzip, deflate, br"
+  "Accept-Encoding" = "gzip, deflate, br, zstd"
   "X-Custom-Header-1" = "Custom value"
   "X-Custom-Header-2" = "8.8.8.8"
   "X-Custom-Header-3" = "Mon, 3 Mar 2014 11:11:11 GMT"
@@ -42,7 +42,7 @@
 -Headers @{
 "Accept" = "*/*"
   "Accept-Language" = "en-US"
-  "Accept-Encoding" = "gzip, deflate, br"
+  "Accept-Encoding" = "gzip, deflate, br, zstd"
   "X-Custom-Header-1" = "Custom value"
   "X-Custom-Header-2" = "8.8.8.8"
   "X-Custom-Header-3" = "Mon, 3 Mar 2014 11:11:11 GMT"
@@ -66,7 +66,7 @@
 -Headers @{
 "Accept" = "*/*"
   "Accept-Language" = "en-US"
-  "Accept-Encoding" = "gzip, deflate, br"
+  "Accept-Encoding" = "gzip, deflate, br, zstd"
   "X-Custom-Header-1" = "Custom value"
   "X-Custom-Header-2" = "8.8.8.8"
   "X-Custom-Header-3" = "Mon, 3 Mar 2014 11:11:11 GMT"
@@ -95,7 +95,7 @@
 -Headers @{
 "Accept" = "*/*"
   "Accept-Language" = "en-US"
-  "Accept-Encoding" = "gzip, deflate, br"
+  "Accept-Encoding" = "gzip, deflate, br, zstd"
   "X-Custom-Header-1" = "Custom value"
   "X-Custom-Header-2" = "8.8.8.8"
   "X-Custom-Header-3" = "Mon, 3 Mar 2014 11:11:11 GMT"